userscript/lovely-forks.user.js

```diff
@@ -0,0 +1,501 @@
+// ==UserScript==
+// @name         Lovely Forks
+// @namespace    musically-ut
+// @version      2.6.2
+// @description  Show notable forks of GitHub projects.
+// @homepage     https://github.com/musically-ut/lovely-forks
+// @icon         https://github.com/musically-ut/lovely-forks/raw/master/userscript/icon.png
```

Comment:
No, just for semantics and if you change the icon path or to different. I expected the app/../ icon's path will be changed since WebExtensions, and the userscript may not be updated, causing the icon to break.